HPC/README.md

```diff
@@ -184,6 +186,11 @@ Thread model: posix
 InstalledDir: /shared/arm/arm-linux-compiler-23.04_Ubuntu-20.04/bin
 ```
 
+### Storage
+Some HPC applications require significant amounts of file I/O, however HPC instance types (Graviton instances included) don't have local storage, and have limited EBS bandwidth and IOPS.  Relying on EBS on each node can cause surprise slow-downs when the instance runs out of EBS burst credits.  This is one reason we don't recommend using an hpc7g (or any HPC instance type) for headnodes, since the headnode performs additional I/O as the scheduler, and often serves a home directory to the compute nodes.  For these reasons the following recommendations are made:
```

Comment:
```suggestion
Some HPC applications require significant amounts of file I/O, however HPC instance types (Graviton instances included) don't have local storage, and have limited EBS bandwidth and IOPS.  Relying on EBS on each node can cause surprise slow-downs when the instance runs out of EBS burst credits.  This is one reason we don't recommend using an HPC7g (or any HPC instance type) for headnodes, since the headnode performs additional I/O as the scheduler, and often serves a home directory to the compute nodes.  For these reasons the following recommendations are made:
```

The AWS documentation styles this as "Hpc7g" but I noticed we write it in this document as "HPC7g". We should probably be consistent within the document. Then that's all from me.